docs/api-reference/next/image.md

```diff
@@ -118,78 +154,82 @@ const MyImage = (props) => {
+- Second, the `sizes` property configures how `next/image` automatically generates an image source set. If no `sizes` value is present, a small source set is generated, suitable for a fixed-size image. If `sizes` is defined, a large source set is generated, suitable for a responsive image. If the `sizes` property includes sizes such as `50vw`, which represent a percentage of the viewport width, then the source set is trimmed to not include any values which are too small to ever be necessary.
 
 For example, if you know your styling will cause an image to be full-width on mobile devices, in a 2-column layout on tablets, and a 3-column layout on desktop displays, you should include a sizes property such as the following:
 
-```js
+```jsx
 import Image from 'next/image'
-const Example = () => (
-  <div className="grid-element">
+
+export default function Page() {
+  ;<div className="grid-element">
     <Image
       src="/example.png"
-      fill
-      sizes="(max-width: 768px) 100vw,
-              (max-width: 1200px) 50vw,
-              33vw"
+      layout="fill"
+      sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
     />
   </div>
-)
+}
 ```
 
 This example `sizes` could have a dramatic effect on performance metrics. Without the `33vw` sizes, the image selected from the server would be 3 times as wide as it needs to be. Because file size is proportional to the square of the width, without `sizes` the user would download an image that's 9 times larger than necessary.
 
-Learn more about `srcset` and `sizes`:
+Learn more about [`srcset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-srcset) and [`sizes`](https://web.dev/learn/design/responsive-images/#sizes).
 
-- [web.dev](https://web.dev/learn/design/responsive-images/#sizes)
-- [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes)
+#### `quality`
 
-### quality
+```js
+quality={75} // {number 1-100}
+```
 
 The quality of the optimized image, an integer between `1` and `100`, where `100` is the best quality and therefore largest file size. Defaults to `75`.
 
-### priority
+#### `priority`
+
+```js
+priority={false} // {false} | {true}
+```
 
 When true, the image will be considered high priority and
 [preload](https://web.dev/preload-responsive-images/). Lazy loading is automatically disabled for images using `priority`.
 
 You should use the `priority` property on any image detected as the [Largest Contentful Paint (LCP)](https://nextjs.org/learn/seo/web-performance/lcp) element. It may be appropriate to have multiple priority images, as different images may be the LCP element for different viewport sizes.
 
-Should only be used when the image is visible above the fold. Defaults to `false`.
+The `priority` property should only be used when the image is visible above the fold. Defaults to `false`.
 
-### placeholder
+#### `placeholder`
+
+```js
+placeholder = 'empty' // {empty} | {blur}
```

Comment:
I'm not sure I like the curly brackets here. How about confirming to TS syntax and using this:

```suggestion
placeholder = 'empty' // "empty" | "blur"
```